build/mvn

```diff
@@ -69,7 +69,7 @@ install_app() {
 
 # Install maven under the build/ folder
 install_mvn() {
-  local MVN_VERSION="3.3.3"
+  local MVN_VERSION="3.3.9"
```

Comment:
If we're going to bump Maven, should we also make sure that we have an enforcer rule which doesn't allow any earlier version of Maven to be used?